manifests/concourse-manifest/extensions/datadog-agent.yml

```diff
@@ -1,8 +1,8 @@
 releases:
   - name: riemann
-    url: https://github.com/alphagov/paas-cf/releases/download/riemann-1/riemann-gds-1.tgz
-    sha1: 8a4220a039ce6a2a0383aab8d4a76746501e58ec
-    version: gds-1
+    url: https://s3-eu-west-1.amazonaws.com/gds-dev-boshreleases/riemann-gds-4.tgz
```

Comment:
I have just uploaded the 149MB file as a GitHub release (and then deleted it straight after as it was attached to some random tag) on the paas-riemann-boshrelease repository and it was allowed. We should store this release on GitHub, because our own self-managed release process is not mature enough.